src/Pelagos/Bundle/AppBundle/Resources/public/js/data-discovery.js

```diff
@@ -360,9 +376,9 @@ function showDatasets(by,id) {
                     loadData(by, id);
                 }
             });
-            if (myGeoViz.getFilter()) {
+            if (geo_filter) {
                 $("#clearGeoFilterButton").button("enable");
-            }
+             }
```

Comment:
Remove space.

Reply:
I'll fix it in feature/PELAGOS-3319 (the last subtask)